btk/measure.py

```diff
@@ -135,18 +137,107 @@ def basic_measure(
     return {"catalog": catalog}
 
 
-def sep_measure(
+def sep_multiband_measure(
     batch,
     idx,
     channels_last=False,
     surveys=None,
+    matching_threshold=1.0,
     sigma_noise=1.5,
     is_multiresolution=False,
     **kwargs,
 ):
-    """Return detection, segmentation and deblending information with SEP.
+    """Returns centers detected with source extractor by combining predictions in different bands.
+
+    NOTE: If this function is used with the multiresolution feature,
+    measurements will be carried on the first survey.
```

Comment:
This is somethign about this function that is somewhat implicit and may surprise users, please let me know if you have any ideas how to improve this or if you think this is OK ?

Reply:
Hmm, I am not sure what would be the best approach here. 

This is how the original `sep_measure` worked on multiresolution input, so I just decided to copy this part into `sep_multiband_measure` function. It also implemented the same way in other measurement functions like `basic_measure`. 

Should we change this to iteration over all surveys?

Reply:
Thanks @atorchylo I realized my original comment was a bit vague. I was thinking of something simpler, like allowing the user to specify explicitly which of the surveys they want to use in the case of MR. 

This is part of a broader issue I see with the MR functionality that is currently implemented, we can discuss this more sometime if you are interested but I think this is out of scope of this PR.